--- src/settings/user.rs
+++ src/settings/user.rs
@@ -2291,14 +2291,14 @@
         if let Some(error) = &self.error_message {
             col = col.push(settings::section().add(settings::item(
                 error.as_str(),
-                button::text("Dismiss").on_press(Message::DismissError),
+                button::text(crate::fl!("dismiss")).on_press(Message::DismissError),
             )));
         }

         if let Some(msg) = &self.success_message {
             col = col.push(settings::section().add(settings::item(
                 msg.as_str(),
-                button::text("Dismiss").on_press(Message::DismissSuccessMessage),
+                button::text(crate::fl!("dismiss")).on_press(Message::DismissSuccessMessage),
             )));
         }
