--- src/settings/user.rs
+++ src/settings/user.rs
@@ -2104,18 +2104,18 @@

             if !status.is_complete() {
                 let mut btn = button::text(if self.is_bootstrapping {
-                    "Bootstrapping..."
+                    crate::fl!("bootstrapping")
                 } else {
-                    "Bootstrap Cross-signing"
+                    crate::fl!("bootstrap-cross-signing")
                 });
                 if !self.is_bootstrapping {
                     btn = btn.on_press(Message::BootstrapCrossSigning);
                 }
                 section = section.add(settings::item_row(vec![btn.into()]));
             }
         } else {
             let mut btn = button::text(if self.is_bootstrapping {
-                "Bootstrapping..."
+                crate::fl!("bootstrapping")
             } else {
-                "Setup Cross-signing"
+                crate::fl!("setup-cross-signing")
             });
             if !self.is_bootstrapping {
                 btn = btn.on_press(Message::BootstrapCrossSigning);
