--- src/settings/user.rs
+++ src/settings/user.rs
@@ -2188,7 +2188,7 @@
                         text_input("Phone Number", &self.new_3pid_msisdn)
                             .on_input(Message::New3PIDMsisdnChanged),
                     )
-                    .push(button::text("Send SMS").on_press(Message::Request3PIDMsisdnToken))
+                    .push(button::text(crate::fl!("send-sms")).on_press(Message::Request3PIDMsisdnToken))
                     .wrap(),
             ));
         }
@@ -2205,7 +2205,7 @@
             for user_id in &self.ignored_users {
                 section = section.add(settings::item(
                     user_id.as_str(),
-                    button::text("Unignore").on_press(Message::UnignoreUser(user_id.clone())),
+                    button::text(crate::fl!("unignore")).on_press(Message::UnignoreUser(user_id.clone())),
                 ));
             }
